Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -567,15 +568,18 @@ protected class ClassWriter : IClassWriter {
     protected override void EmitLiteralExpr(ConcreteSyntaxTree wr, LiteralExpr e) {
       switch (e) {
         case CharLiteralExpr:
-          wr.Write("'{0}'", (string)e.Value);
+          wr.Write($"'{(string)e.Value}'");
```

Comment:
Do you need the cast? (And, does `Write` correctly support the `0`-arguments case, or will it break if `e.Value` contains curly braces?)

Reply:
The `Value` is a raw pointer (object), so it seems sensible here. What's the 0-arguments case? `""`?
This works:
```Dafny
method Main(){
  var x := "{";
  print x, "\n";
}
```